docs/sdk/authentication/session-sigs/get-session-sigs.md

```diff
@@ -3,112 +3,256 @@ sidebar_position: 2
+
+### `capabilityAuthSigs`
+Not limited to capacityDelegationAuthSig, we want to be able to pass in any other authSigs for other purposes.
+
+### `authNededCallback`
+This is a callback that will be called if you need to authenticate using a PKP. If you don't pass this callback, then the user will be prompted to authenticate with their wallet, like MetaMask.
+
+## Return Value
+
+### `sig`
+The signature produced by the ed25519 key pair signing the `signedMessage` payload.
+
+### `derivedVia`
+Should be `litSessionSignViaNacl`, specifies that the session signature object was created via the `NaCl` library.
+
+### `signedMessage`
+The payload signed by the session key pair. This is the signed `AuthSig`, which includes the fields:
+
+- #### `sessionKey`
+   The session key pair public key.
+- #### `resourceAbilityRequests`
+   An array of resource abilities that you have requested for the session.
+- #### `capabilities` <- Needs a fact check
+   An array of one or more `AuthSig`. The `capabilities` authorize the `AuthSig` address(es) to utilize the resources specified. in the `resourceAbilityRequests`
+- #### `issuedAt`
+   The time the session signature was issued.
+- #### `expiration`
+   The time the session signature becomes invalid.
+- #### `nodeAddress`
+   The specific URL the session signature is meant for.
+
+### `address`
+The session key pair public key.
+
+### `algo`
+The signing algorithm used to generate the session signature.
+
+## Example Implementation
+
+Now that we know what the `getSessionSigs` function does, it's parameters, and it's return values, let's dig into a complete implementation.
+
+The full code implementation can be found [here](https://github.com/LIT-Protocol/developer-guides-code/tree/master/session-signatures/getSessionSigs). 
+
+### Installing the Required Dependencies
+<Tabs
+defaultValue="npm"
+values={[
+{label: 'npm', value: 'npm'},
+{label: 'yarn', value: 'yarn'},
+]}>
+<TabItem value="npm">
+
+```bash
+npm install \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+node-localstorage \
+ethers@v5
+```
+
+</TabItem>
+
+<TabItem value="yarn">
+
+```bash
+yarn add \
+@lit-protocol/auth-helpers \
+@lit-protocol/constants \
+@lit-protocol/lit-node-client \
+node-localstorage \
+ethers@v5
+```
+
+</TabItem>
+</Tabs>
+
+
+
+### Instantiating an Ethers Signer
+The `ETHEREUM_PRIVATE_KEY` environment variable is required.
+```ts
+import { LIT_RPC } from "@lit-protocol/constants";
+import * as ethers from "ethers";
+
+const ethersSigner = new ethers.Wallet(
+  ETHEREUM_PRIVATE_KEY,
+  new ethers.providers.JsonRpcProvider(LIT_RPC.CHRONICLE_YELLOWSTONE)
+);
+```
+
+### Instantiating a `LitNodeClient`
+Here we are instantiating an instance of `LitNodeClient` and connecting it to the `datil-dev` Lit network. To simulate a Node.js environment, we use `LocalStorage` from the `node-localstorage` module to store our session keys and metadata.
```

Comment:
```suggestion
Here we are instantiating an instance of `LitNodeClient` and connecting it to the `datil-dev` Lit network. Because we're executing within a Node.js environment, we use `LocalStorage` from the `node-localstorage` module to store our session keys and metadata.
```